tiledb/sm/query/reader.cc

```diff
@@ -1282,6 +1282,12 @@ Status Reader::copy_partitioned_var_cells(
       tile_cell_num = tile->cell_num();
     }
 
+    // Fetch the offsets format from config
```

Comment:
This `copy_partitioned_var_cells` is a performance-critical routine, so we need to fetch this somewhere else and cache it. For now, let's fetch this in `Reader::init` and store a `std::string offsets_format_` as a member variable. While we're there, we should do a run-time check that the value is either "bytes" or "elements" and return a non-OK status if not.

Reply:
sure, I didn't know it was a performance-critical path